pkg/ast/pipesearch/searchHandler.go

```diff
@@ -482,6 +500,17 @@ func convertRRCsToJSONResponse(rrcs []*sutils.RecordResultContainer, sizeLimit u
 	if sizeLimit < uint64(len(allJsons)) {
 		allJsons = allJsons[:sizeLimit]
 	}
+
+	if !includeNulls {
+		for _, record := range allJsons {
+			for key, value := range record {
+				if value == nil {
+					delete(record, key)
+				}
+			}
```

Comment:
This should work for the current pipeline, but I don't think the new pipeline calls this function (I might be wrong). Can you check?

Reply:
It does work for the new pipeline. Could you please let me know where I need to make changes for the new pipeline?